src/session.rs

```diff
@@ -27,8 +28,13 @@ const NACK_MIN_INTERVAL: Duration = Duration::from_millis(100);
 // Delay between reports of TWCC. This is deliberately very low.
 const TWCC_INTERVAL: Duration = Duration::from_millis(100);
 
-const PACING_FACTOR: f64 = 1.1;
-const PADDING_FACTOR: f64 = 1.0;
+// The target bitrate must be slightly above an estimated bitrate
+// so that the pacer tries to "push upwards" when below the
+// desired bitrate.
+const PADDING_OVER: Bitrate = Bitrate::kbps(5);
```

Comment:
Yeah. Basically a %-based constant means we get these crazy padding additions the higher we are. That's not good. Say our bandwidth cap is 3mbit. As we approach it, we pad like +/- 300kbit at a time. This will likely overshoot and overcompensate down again.

Reply:
> Yeah. Basically a %-based constant means we get these crazy padding additions the higher we are.

Citation needed. I don't think this was ever happening before. In fact, the padding factor was previously set to `1.0` which meant we never exceeded the estimate with padding. If anything this change is making it ever so slightly worse

Also the comment doesn't correctly reflect how the system works. 

What actually happens is that the `RateControl` keeps increasing the estimate as long as we aren't detecting overuse, this happens regardless of the padding. The purpose of the padding is to ensure that the estimate isn't a lie i.e. that if we estimate 300Kbit/s we can actually be confident in that estimate because we have been sending 300Kbit/s.

Reply:
Right. So if I get this right, you're saying there's another mechanism that increases the estimate, and this extra padding on top is not necessary. That's fine. I thought one had to attempt to send more than current estimate to actually get a higher estimate.

Sounds like we can remove the padding constant completely.

Reply:
The idea is that the padding + media should reach the estimate so we have confidence in our estimate. I think setting the target/padding rate to the estimate or slightly below is fine.